datafusion/common/src/error.rs

```diff
@@ -493,16 +500,16 @@ macro_rules! make_error {
 }
 
 // Exposes a macro to create `DataFusionError::Plan`
-make_error!(plan_err, Plan);
+make_error!(plan_err, plan_err_raw, Plan);
```

Comment:
I was not able to invent better naming, appreciate if anyone helps

Reply:
Maybe `plan_err_obj` or `plan_datafusion_error`?